docs/getting-started/install/linux/index.md

```diff
@@ -12,7 +12,7 @@ Regardless of your distribution, you will need the Mono Project GPG signing key,
 Debian, Ubuntu, and derivatives
```

Comment:
I'd like to give it a shot. I just need a modern jekyll installed to test a local gh-pages branch, right?